lint_test/avoid_final_with_getter_test.dart

```diff
@@ -0,0 +1,28 @@
+// ignore_for_file: type_annotate_public_apis, prefer_match_file_name, unused_local_variable
+
+/// Check final private field with getter fail
+/// `avoid_final_with_getter`
+
+class Fail {
+  final int _myField = 0;
+
+  // expect_lint: avoid_final_with_getter
+  int get myField => _myField;
+}
+
+class Fail2 {
+  final int _myField = 0;
+
+  // expect_lint: avoid_final_with_getter
+  int get myFieldInt => _myField;
+}
```

Comment:
Does it make sense to add a test for the static field and getter?

Reply:
yes, it does